src/renderer/src/helpers/PasswordHelper.ts

```diff
@@ -44,4 +45,67 @@ export class PasswordHelper {
         /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[!@#$%^&*()_+[\]{};':"\\|,.<>/?]).+$/.test(password))
     )
   }
+
+  static #getRandomLower() {
+    let result = ''
+    const characters = 'abcdefghijklmnopqrstuvwxyz'
+    const charactersLength = characters.length
+    let counter = 0
+    while (counter < length) {
+      result += characters.charAt(Math.floor(Math.random() * charactersLength))
+      counter += 1
+    }
+    return result
+  }
+
+  static #getRandomUpper() {
+    let result = ''
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    const charactersLength = characters.length
+    let counter = 0
+    while (counter < length) {
+      result += characters.charAt(Math.floor(Math.random() * charactersLength))
+      counter += 1
+    }
+    return result
+  }
+
+  static #getRandomSymbol() {
+    let result = ''
+    const characters = '!@#$%^&*()_+[\\]{};\':"\\|,.<>/?'
+    const charactersLength = characters.length
+    let counter = 0
+    while (counter < length) {
+      result += characters.charAt(Math.floor(Math.random() * charactersLength))
+      counter += 1
+    }
+    return result
+  }
+
+  static generateStrongPassword() {
```

Comment:
It's not working well.
<img width="533" height="262" alt="Screenshot 2025-12-30 at 17 16 28" src="https://github.com/user-attachments/assets/c81172fc-b36f-4750-86f6-d2632ad27976" />